R/LearnerRegrGAMBoost.R

```diff
@@ -0,0 +1,86 @@
+#' @title Boosted Generalized Additive Regression Learner
+#'
+#' @aliases mlr_learners_regr.gamboost
+#' @format [R6::R6Class] inheriting from [LearnerRegr].
+#'
+#' @description
+#' A [LearnerRegr] for a regression gamboost implemented in [mboost::gamboost()] in package \CRANpkg{mboost}.
+#'
+#' @references
+#' Peter Buhlmann and Bin Yu (2003)
+#' Boosting with the L2 Loss: Regression and Classification
+#' Journal of the American Statistical Association
+#' \url{https://doi.org/10.1198/016214503000125}
+#'
+#' @export
+LearnerRegrGAMBoost = R6Class("LearnerRegrGAMBoost", inherit = LearnerRegr,
+  public = list(
+    initialize = function() {
+      ps = ParamSet$new(
+        params = list(
+          ParamFct$new(id = "baselearner", default = "bbs", levels = c("bbs", "bols", "btree"), tags = c("train")),
+          ParamInt$new(id = "dfbase", default = 4L, tags = c("train")),
+          ParamDbl$new(id = "offset", default = NULL, special_vals = list(NULL), tags = c("train")),
+          ParamFct$new(id = "family", default = c("Gaussian"),
```

Comment:
Try to avoid concatenating scalar values.

Reply:
Forgot to change this after removing some tags